src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php

```diff
@@ -330,7 +358,10 @@ private static function addExtra(&$a, \Reflector $c)
         }
     }
 
-    private static function addMap(&$a, \Reflector $c, $map, $prefix = Caster::PREFIX_VIRTUAL)
+    /**
+     * @param \Reflector|\ReflectionAttribute $c
+     */
+    private static function addMap(array &$a, $c, $map, $prefix = Caster::PREFIX_VIRTUAL)
```

Comment:
💡 It felt a bit odd that I needed to do this, so I've opened https://bugs.php.net/bug.php?id=80097 for clarification.